protocol/0057-TRAN-transfers.md

```diff
@@ -155,6 +155,8 @@ Note: if there is no market with contribution to the reward metric - no transfer
 
 A fee is taken from all transfers, and paid out to validators in a similar manner to the existing [infrastructure fees](0061-REWP-pos_rewards.md). For recurring transfers, the fee is charged each time the transfer occurs.
 
+For each party, we aggregate the fees paid during an epoch and maintain a running total. If there is a transfer activity, we deduct the corresponding fee from the accumulated total. Once this total reaches zero, the party incurs charges. We require API support to enable the frontend to transparently display the amount eligible for fee-free transfers.
```

Comment:
1) Which fees? (Taker fees)
2) Does this apply only per-asset basis? Ie if I pay 0.5USDT in fees and then make a USDT transfer I get a discount. OR we use quantum to convert? I think per asset is ok but needs to be clear.

Reply:
3. Is this counted during the epoch I.e during epoch N I spend 0.5 on taker fees then in the same epoch I make a transfer and get a discount OR during epoch N I pay taker fees and then during epoch N+1 I can get a discount? OR we ignore epochs and just cumulate all taker fees paid across key lifetime and use this for discounts?

Reply:
1. Any trading fees - so if you meam the full taker fee (infra+maker+LP components) then yes
2. Per asset seems fine
3. I guess either lifetime or last N (7? 30?) epochs?